src/bioregistry/data/bioregistry.json

```diff
@@ -22211,6 +22211,58 @@
     "name": "Crates.io",
     "uri_format": "https://crates.io/crates/$1"
   },
+  "creativebiolabs.antibody": {
+    "contact": {
+      "email": "info@creative-biolabs.com",
+      "name": "Creative Biolabs Team"
```

Comment:
@garybrandam the Bioregistry expects a specific point of contact person for each prefix. Could you provide contact info for an appropriate person, ideally name, ORCID, email, GitHub handle if available?